syntax = "proto3";


option java_multiple_files = true;
option java_package = "io.grpc.examples.helloworld";
option java_outer_classname = "HelloWorldProto";
option objc_class_prefix = "HLW";

package helloworld;


service Greeter{
  rpc SayHello(HelloRequest) returns (HelloResponse){};
  rpc SayHelloAgain(HelloRequest) returns (HelloResponse){};

}


message HelloRequest{
  string name = 1;
}

message HelloResponse{
  string message = 1;
  int32 count=2;
}